apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationDialog.tsx

```diff
@@ -0,0 +1,429 @@
+import { useIntl } from "react-intl";
+import { differenceInDays } from "date-fns/differenceInDays";
```

Comment:
`differenceInDays` is defined but never used.
```suggestion
```

Reply:
Fixed in [5ddc989](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/5ddc989c3b33933fccfba9a2af5672e6b429b038)